core_v2/sources/domains.move

```diff
@@ -156,7 +146,114 @@ module aptos_names::domains {
             register_name_events: account::new_event_handle<RegisterNameEventV1>(account),
         });
 
-        token_helper::initialize(account);
+        // Create collection + token_resource
+        let registry_seed = utf8_utils::u128_to_string((timestamp::now_microseconds() as u128));
+        string::append(&mut registry_seed, string::utf8(b"registry_seed"));
+        let (token_resource, token_signer_cap) = account::create_resource_account(
+            account,
+            *string::bytes(&registry_seed),
+        );
+        move_to(account, CollectionCapabilityV2 {
+            capability: token_signer_cap,
+        });
+        collection::create_unlimited_collection(
+            &token_resource,
```

Comment:
in the [hero example](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/token_objects/hero/sources/hero.move#L67) it just use the account as creator, does that mean the collection will be owned directly by the account? and in our case we create a resource account under owner and put the collection in resource account?